jdaviz/configs/cubeviz/helper.py

```diff
@@ -41,24 +41,28 @@ def _set_spectrum_x_axis(self, msg):
                 viewer.state.x_att = ref_data.id["Pixel Axis 2 [x]"]
                 viewer.state.reset_limits()
 
-    def load_data(self, data, data_label=None, **kwargs):
+    def load_data(self, data, data_label=None, override_cube_limit=False, **kwargs):
         """
         Load and parse a data cube with Cubeviz.
         (Note that only one cube may be loaded per Cubeviz instance.)
 
         Parameters
         ----------
-        data : str, `~astropy.io.fits.HDUList`, or `~specutils.Spectrum1D`
+        data : str, `~astropy.io.fits.HDUList`, `~specutils.Spectrum1D`, or ndarray
             A string file path, astropy FITS object pointing to the
-            data cube, or a spectrum object.
+            data cube, a spectrum object, or a Numpy array cube.
         data_label : str or `None`
             Data label to go with the given data. If not given,
             one will be automatically generated.
+        override_cube_limit : bool
+            Override internal cube count limitation and load the data anyway.
+            Setting this to `True` is not recommended unless you know what
+            you are doing.
```

Comment:
is this needed for the tests?  Since we already have `**kwargs`, could we abstract it away there so the user doesn't know about it?

Reply:
@kecnry , without this override, you cannot load the flux cube first as plain array, and then load the uncertainty later (also plain array). This is needed to load things into Cubeviz but they are not all different components of the same cube.

I could hide this in kwargs, yes, but that is more code to then extract it back out and throw it away before passing kwargs on. Is that really what you want?

Reply:
I don't know - I agree that isn't ideal either, but this is a fairly public and widely used method so I'm trying to think of ways to avoid an "advanced" parameter that overrides this check.  What if we either enforced a single flux cube, or allowed loading numpy arrays, but not additional fits or specutils cubes instead?

Reply:
> What if we either enforced a single flux cube, or allowed loading numpy arrays, but not additional fits or specutils cubes instead?

That is too complicated. Really, people should not be using this option unless they really know what they are doing... 😆 

So hiding in kwargs then?

Reply:
So then they should not be loading arrays?  Is this just implemented for internal testing?  If so, then I'd vote to hide it in kwargs, but am curious what others think.

Reply:
I mean for the 5 people in the world who really know how to use Cubeviz, this would be a great feature. You can load "flux" from one file, "uncertainty" from another file (really, this can just be any second cube), display them side by side and do stuff.